aws/ec2.go

```diff
@@ -0,0 +1,91 @@
+package aws
+
+import (
+	"fmt"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/aws-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// Build string that'll be shown in resources list
+func buildEntryName(instance ec2.Instance) string {
+	return fmt.Sprintf(
+		"ec2-%s-%s",
+		awsgo.StringValue(instance.InstanceId),
+		awsgo.StringValue(instance.InstanceType),
+	)
+}
+
+// Returns a formatted string of EC2 instance ids
```

Comment:
Update the comment, as these IDs are not formatted. Also, mention this function returns just instances that are not already terminated and not protected.

Reply:
I'm removing this function as it's no longer being used. My last refactor removed the need for it, just forgot to delete it